### PR TITLE
feat: Implement UpdatableAdapter on adapter

### PR DESCRIPTION
**Pull request by jkalberer**
This updates the adapter so that we can use updatePolicy from the enforcer. 
I also set up local testing to use sqlite so we can run the tests without a mysql instance.

### CODE REVIEW

**Comment by jkalberer** [comment]
Sorry for the spam on the other PRs -- I had to make sure CI passed and did't want to deal with fixing the commits :)

File: package.json [hunk 1]
```diff
@@ -9,8 +9,8 @@
     "prepublish": "yarn run lint && yarn build",
     "build": "rimraf lib && tsc",
     "coverage": "jest --coverage --runInBand",
-    "lint": "eslint src/**/*.{ts,tsx,js,jsx}",
-    "fix": "eslint src/**/*.{ts,tsx,js,jsx} --fix",
```
Comment (jkalberer) [inline package.json:13]: This was failing for me on my mac for some reason...?

**Comment by hsluoyz** [comment]
You should fix your commits in existing PR, not creating new PRs

**Comment by jkalberer** [comment]
@hsluoyz - yep, apologies. I didn't know I could just amend and force push. Will do so in the future.

**Pull request merged**